File: BI-TJV/semestral/rest-server/src/main/java/wrzecond/entity/TjvTable.java
package wrzecond.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.SequenceGenerator;

import lombok.*;
import wrzecond.table.TjvTableType;

@Entity
@EqualsAndHashCode(callSuper = true)
@AllArgsConstructor
@NoArgsConstructor
public class TjvTable extends TjvEntity {

    @Column(nullable = false)
    @Getter @Setter private TjvTableType type;

}

File: BI-KOT/semestral/rest-server/src/main/kotlin/wrzecond/entity/TjvTable.kt
package wrzecond.entity

import wrzecond.TjvTableType
import javax.persistence.Column
import javax.persistence.Entity

@Entity
data class TjvTable (
    @Column(nullable = false) val type: TjvTableType,
    override val id: Int = 0
) : TjvEntity()
